src/components/v5/frame/ColonyHome/partials/DashboardHeader/DashboardHeader.tsx

```diff
@@ -28,7 +28,7 @@ const DashboardHeader = () => {
           />
         )}
       </div>
-      <div className="flex flex-col gap-4 sm:flex-row sm:gap-7">
+      <div className="flex flex-col gap-3 sm:flex-row sm:gap-7">
```

Comment:
This looks like it should be gap-4 to me? Not sure if there is a design inconsistency somewhere?

<img width="637" alt="Screenshot 2024-10-04 at 12 59 52" src="https://github.com/user-attachments/assets/1615f131-212d-42dc-9699-f0481ea2ac48">

<img width="401" alt="Screenshot 2024-10-04 at 13 00 00" src="https://github.com/user-attachments/assets/b372cc80-d9af-43a2-9b51-3aa7df8c7124">

Reply:
reverted, thanks!